### PR TITLE
Feat gallery lightbox

### PR DESCRIPTION
**Pull request by mixcas**
Add lightbox gallery feature to news
screen shot 2016-12-20 at 8 46 45 pm 2

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -421,6 +423,7 @@ Site.Lightbox = {
 
     $('body').addClass('lightbox-active');
     _this.$lightbox.css('display', 'flex');
+    Site.Galleries.init();
```
Comment (varffs) [inline js/main.js]: Does this retrigger existing galleries? Does this matter?

File: lib/thumbnail-sizes.php [hunk 2]
```diff
@@ -25,6 +25,8 @@
   add_image_size( 'l-8-square', 866, 866, true );
   add_image_size( 'l-8-full', 866, 9999, false );
 
+  add_image_size( 'lightbox', 1030, 644, false );
```
Comment (varffs) [inline lib/thumbnail-sizes.php]: how is this size smaller the `l-12`? I thought that was the reference size?

File: index.php [hunk 3]
```diff
@@ -8,15 +8,19 @@
...
 
 <?php
-if( have_posts() ) {
+if ( have_posts() ) {
   while( have_posts() ) {
     the_post();
     $explode_content = explode( '<!--more-->', $post->post_content );
     $lightbox_image = wp_get_attachment_image_src(get_post_thumbnail_id(), 'lightbox');
+
+    $gallery = get_post_meta($post->ID, '_igv_gallery', true);
```
Comment (varffs) [inline index.php]: Need to remove the thumbnail loading and just do gallery if exists. But maybe not scope of feature branch?

Comment (varffs) [inline index.php]: Yh the thumbs no longer load in the lightbox it says in the card. Only the gallery loads if there is one (how the user is supposed to know I've no idea)

Comment (mixcas) [inline index.php]: Ohhh ok

File: js/main.js (hunk 1 above)
Comment (mixcas) [inline js/main.js]: Yeah,  i'll fix it.

File: lib/thumbnail-sizes.php (hunk 2 above)
Comment (mixcas) [inline lib/thumbnail-sizes.php]: I don't remember cuz it didn't fit the screen vertically, neither on my monitor nor in the macbook screen. So the image would almost always end up scaling down anyway. 

I used 10 col width.

Comment (varffs) [inline lib/thumbnail-sizes.php]: Ok lets see how it is but surely the oversize should be solvable with CSS? I think there is a class that I styled for the thumbnail that could be used?

**Comment by mixcas** [comment]
I didn't remove the functionality to open single images on the lightbox. I consider it doesn't harm to keep it around for a bit.

**Pull request merged**